modeshape-jcr/src/main/resources/org/modeshape/jcr/repository-config-schema.json

```diff
@@ -553,6 +553,28 @@
                 },
             }
         },
+        "clustering": {
+            "type": "object",
+            "description": "The clustering configuration. If absent, clustering will be disabled",
+            "additionalProperties": true,
```

Comment:
What additional properties are allowed/supported? Perhaps the `additionalProperties` value should be `false`.